crates/rpc-types-engine/src/sidecar.rs

```diff
@@ -28,45 +28,45 @@ impl OpExecutionPayloadSidecar {
     /// Extracts the [`OpExecutionPayloadSidecar`] from the given [`Block`].
     ///
     /// Returns `OpExecutionPayloadSidecar::default` if the block does not contain any sidecar
-    /// fields (pre-canyon).
+    /// fields (pre-ecotone).
     pub fn from_block<T, H>(block: &Block<T, H>) -> Self
     where
         T: Transaction,
         H: BlockHeader,
     {
-        let canyon =
+        let ecotone =
             block.parent_beacon_block_root().map(|parent_beacon_block_root| CancunPayloadFields {
                 parent_beacon_block_root,
                 versioned_hashes: block.body.blob_versioned_hashes_iter().copied().collect(),
             });
 
         let isthmus = block.requests_hash().map(PraguePayloadFields::new);
 
-        match (canyon, isthmus) {
-            (Some(canyon), Some(isthmus)) => Self::v4(canyon, isthmus),
-            (Some(canyon), None) => Self::v3(canyon),
+        match (ecotone, isthmus) {
+            (Some(ecotone), Some(isthmus)) => Self::v4(ecotone, isthmus),
+            (Some(ecotone), None) => Self::v3(ecotone),
             _ => Self::default(),
         }
     }
 
-    /// Creates a new instance for canyon with the canyon fields for `engine_newPayloadV3`
-    pub fn v3(canyon: CancunPayloadFields) -> Self {
-        Self { canyon: canyon.into(), isthmus: Default::default() }
+    /// Creates a new instance for ecotone with the ecotone fields for `engine_newPayloadV3`
+    pub fn v3(ecotone: CancunPayloadFields) -> Self {
+        Self { ecotone: ecotone.into(), isthmus: Default::default() }
     }
 
     /// Creates a new instance post prague for `engine_newPayloadV4`
-    pub fn v4(canyon: CancunPayloadFields, isthmus: PraguePayloadFields) -> Self {
-        Self { canyon: canyon.into(), isthmus: isthmus.into() }
+    pub fn v4(ecotone: CancunPayloadFields, isthmus: PraguePayloadFields) -> Self {
+        Self { ecotone: ecotone.into(), isthmus: isthmus.into() }
     }
 
     /// Returns a reference to the [`CancunPayloadFields`].
-    pub const fn canyon(&self) -> Option<&CancunPayloadFields> {
-        self.canyon.as_ref()
+    pub const fn ecotone(&self) -> Option<&CancunPayloadFields> {
```

Comment:
these are breaking

Reply:
we care about breaking public api now 😮 i see, i see

sure, can do

Reply:
if we break stuff, it requires more work and slows down rollouts